README.md

```diff
@@ -1,79 +1,199 @@
-## This is a WIP FAQ for Tusky | Frequently asked questions about the Tusky Mastodon client
+# Tusky Frequently Asked Questions
 
-Which will include some experimentation with tools to write this FAQ and sync it with GitHub, by @mal0ki (current tool is StackEdit).
+## Installing Tusky
 
-## FAQ
-**Question:** When will the latest version of Tusky be available on F-Droid?
+### What Tusky release should I use?
 
-**Answer:** F-Droid works differently than Google Play. We as the developers do not upload anything, F-Droid builds the app from source once we tag a release in our code repository. This is super awesome because you are guaranteed to get the exact code from our repo, but also means it takes longer for releases to be available.
+There are three different releases of Tusky.
 
-##
-**Q:** How does Tusky support push notifications?
+- The mainstream release. Unless you've been told otherwise, this is what you want
+- Beta releases. These are released to testers before every mainstream release
+- Nightly releases. These are released every night
```

Comment:
That is not actually true, they are built after every push to develop, which can be once a week or multiple times a day. Maybe we should rename it to "Tusk Test"? That is how the app is called when you install it.

Reply:
I updated the description.